packages/legend-studio/src/components/editor/side-bar/WorkspaceRevisionUpdater.tsx

```diff
@@ -0,0 +1,281 @@
+        </div>
+        <div className="modal__body">
+          <ResizablePanelGroup orientation="vertical">
+            <ResizablePanel size={350} minSize={350}>
+              <div className="panel explorer">
+                <div className="panel__header workspace-revision-updater__changes__header">
+                  <div className="panel__header__title">
+                    <div className="panel__header__title__content workspace-revision-updater__changes__header__title__content">
+                      All Changes
+                    </div>
+                  </div>
+                </div>
+                <div className="panel__content explorer__content__container">
+                  <ResizablePanelGroup orientation="horizontal">
+                    <ResizablePanel size={600} minSize={28}>
+                      <div className="panel workspace-revision-updater__changes__panel">
+                        <div className="panel__header">
+                          <div className="panel__header__title">
+                            <div className="panel__header__title__content">
+                              PENDING CONFLICTS
+                            </div>
+                            <div
+                              className="workspace-revision-updater__changes__panel__title__info"
+                              title="All local changes that have not been yet synced with the server"
+                            >
+                              <FaInfoCircle />
+                            </div>
+                          </div>
+                          <div className="workspace-revision-updater__changes__panel__header__changes-count">
+                            {conflicts.length}
+                          </div>
+                        </div>
+                        <div className="panel__content">
+                          {conflicts
+                            .slice()
+                            .sort((a, b) =>
+                              a.entityName.localeCompare(b.entityName),
+                            )
+                            .map((conflict) => (
+                              <EntityChangeConflictSideBarItem
+                                key={`conflict-${conflict.entityPath}`}
+                                conflict={conflict}
+                                isSelected={isSelectedConflict(conflict)}
+                                openConflict={openConflict(conflict)}
+                              />
+                            ))}
+                        </div>
+                      </div>
+                    </ResizablePanel>
+                    <ResizablePanelSplitter>
+                      <ResizablePanelSplitterLine color="var(--color-dark-grey-100)" />
+                    </ResizablePanelSplitter>
+                    <ResizablePanel minSize={20}>
+                      <div className="panel workspace-revision-updater__changes__panel">
```

Comment:
I don't think we should do this differently than we do the other one: i.e. why do we split this into 2 sub-panels here: resolved vs to-be-resolved ?

Reply:
I think it's more clear about what's happening from the user's perspective. Mirrors what vs code does.